File: Num.py
import psycopg2
def  post(x,y):
    try:
        conexion = psycopg2.connect(
            host = "localhost",
            port = "5432",
            user = "postgres",
            password = "123fgthg",
            dbname = "HWP"
            )
    except:
        print("Sin Conexion Exitosa\n")
            
    cursor = conexion.cursor()
    cursor.execute("INSERT INTO numvocg(palabra, numvoc) VALUES (%s, %s);", (x, y))
    conexion.commit()
    cursor.close()
    conexion.close()
print("Numero de Vocales Totale en una palabra")
strt = str(input("Ingrese su palabra:"))

voc=['a','e','i','o','u']
cont=0

for x in range(0, len(strt)):
    if strt[x]== voc[0]:
        cont=cont+1
    if strt[x]== voc[1]:
        cont=cont+1
    if strt[x]== voc[2]:
        cont=cont+1
    if strt[x]== voc[3]:
        cont=cont+1
    if strt[x]== voc[4]:
        cont=cont+1
print('La palabra '+ strt + " tiene: " +str(cont))
post(strt, cont)
exit(0)

File: Num.m
clc; clear all; close all;
strt = input('Ingrese su palabra:      ','s');
voc=['a','e','i','o','u'];
cant=[0,0,0,0,0];
cont=0;
 for x=1:1:length(strt)
    if strt(x) == voc(1)||strt(x) == 'A'
      cont=cont+1;
    endif
    if strt(x) == voc(2)||strt(x) == 'E'
      cont=cont+1;
    endif
    if strt(x) == voc(3)||strt(x) == 'I'
      cont=cont+1;
    endif
    if strt(x) == voc(4)||strt(x) == 'O'
      cont=cont+1;
    endif
    if strt(x) == voc(5)||strt(x) == 'U'
      cont=cont+1;
    endif
 endfor

fprintf('La cantidad de vocales es es: %d \n', cont);

params= cell(1,2);
params{1,1}=strt;
params{1,2}=cont;

save('Num.txt','strt','cont','-ascii');
fprintf('Vocales Totales: %d \n',cont);

conn = pq_connect(setdbopts('dbname','HWP','host','localhost',
'port','5432','user','postgres','password','123fgthg'));
N=pq_exec_params(conn, "insert into numvocg(palabra, numvoc) values($1,$2);",params); %insertar datos en la tabla